src/AutoMapper/Mappers/ObjectCreator.cs

```diff
@@ -59,7 +71,9 @@ public static object CreateObject(Type type)
                 ? CreateArray(type.GetElementType(), 0)
                 : type == typeof (string)
                     ? null
-                    : DelegateFactory.CreateCtor(type)();
+                    : type.IsDictionaryType()

```

Comment:
Yeah, I think in the long run I'll use something that creates an Expression, not a delegate.